.mergify.yml

```diff
@@ -0,0 +1,101 @@
+pull_request_rules:
+  - name: automatic merge for Dependabot pull request that pass CI
+    conditions:
+      - author=dependabot[bot]
+      - check-success=build-workflow-complete
+    actions:
+      merge:
```

Comment:
can we have it leave DP a comment instead?

Reply:
DP comment?

Reply:
I don't know how smart mergify is, but DP is pretty smart. You can omit the check success thing if you leave a comment, and DP will also rebase the PR (and perhaps regenerate it completely in complex cases e.g. Cargo.lock conflicts).